### PR TITLE
Updated the devicon.css file

### PR DESCRIPTION
**Pull request by theblobscp**

### CODE REVIEW

File: devicon.css [hunk 1]
```diff
@@ -1,1134 +1,2048 @@
...
+    url("fonts/devicon.ttf?f7gwp9") format("truetype"),
+    url("fonts/devicon.woff?f7gwp9") format("woff"),
+    url("fonts/devicon.svg?f7gwp9#devicon") format("svg");
   font-weight: normal;
   font-style: normal;
   font-display: block;
 }
-
-[class^="devicon-"], [class*=" devicon-"] {
-  /* use !important to prevent issues with browser extensions that change fonts */
```
Comment (Panquesito7) [inline devicon.css:14]: Please re-add this comment

File: devicon.css [hunk 2]
```diff
@@ -1,1134 +1,2048 @@
...
+[class*=" devicon-"] {
+  font-family: "devicon" !important;
   speak: never;
   font-style: normal;
   font-weight: normal;
   font-variant: normal;
   text-transform: none;
   line-height: 1;
-
-  /* Better Font Rendering =========== */
```
Comment (Panquesito7) [inline devicon.css:23]: Same here as above.

File: devicon.css [hunk 3]
```diff
@@ -1,1134 +1,2048 @@
...
+.devicon-zig-plain:before {
+  content: "";
 }
 .devicon-raspberrypi-line-wordmark:before {
-  content: "\e9ba";
+  content: "";
 }
 .devicon-raspberrypi-line:before {
-  content: "\e9bb";
+  content: "";
```
Comment (Panquesito7) [inline devicon.css:56]: For me, this just displays a weird emoji/text.
I have no idea what that could be. 🤷‍♂️

image

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: devicon.css (hunk 3 above)
Comment (theblobscp) [reply devicon.css:58]: Same for me, I cant figure out how to fix it, maybe it's just vscode acting up?

Comment (Panquesito7) [reply devicon.css:58]: Please consider fixing that problem. Thanks.

Comment (theblobscp) [reply devicon.css:58]: Yeah I found it, I have to manually fix each one so that will be fun

**Comment by Thomas-Boi** [comment]
Hi @theblobscp,

What is the purpose of this PR 🤔? The `devicon.css` is created automatically by Icomoon and is not made by us. It is then used as a build step to make the final `devicon.min.css`. Thus, we don't need to make it look good or anything similar.

As for the weird emoji/text, that's the actual font code for that font. It's fine either way, you don't have to fix that (as long as the final font is still the same as the old one). You can see them in the minified css file as well.

**Comment by Panquesito7** [comment]
I agree with @Thomas-Boi. Also, if we're going to merge this, we would have to test this expensively, as it makes small changes, but those are changes that could ruin everything or the functionality of Devicon.

**Comment by theblobscp** [comment]
> What is the purpose of this PR

Some classes weren't added, so I added them.

**Comment by Panquesito7** [comment]
> Some classes weren't added, so I added them.

That should be added by the bot itself, not manually.

**Comment by theblobscp** [comment]
So I should close this pr?

**Comment by Panquesito7** [comment]
> So I should close this pr?

@Thomas-Boi?

**Comment by Thomas-Boi** [comment]
@Panquesito7 yes, we will need to close this PR.

@theblobscp for future reference, I'll recommend double-checking with us what a file is supposed to do. It will save you time from making unnecessary changes 😅

**Pull request closed without merging**